examples/matRad_example13_fitAnalyticalParticleBaseData.m

```diff
@@ -124,25 +128,44 @@
 pln.propOpt.runDAO          = false;  % 1/true: run DAO, 0/false: don't / will be ignored for particles
 pln.propOpt.runSequencing   = false;  % 1/true: run sequencing, 0/false: don't / will be ignored for particles and also triggered by runDAO below
 
+% retrieve scenarios for dose calculation and optimziation
+pln.multScen = matRad_multScen(ct,'nomScen');
+
+
+quantityOpt   = 'RBExD';            % either  physicalDose / effect / RBExD
+modelName     = 'constRBE';         % none: for photons, protons, carbon                                    constRBE: constant RBE model
+                                    % MCN: McNamara-variable RBE model for protons                          WED: Wedenberg-variable RBE model for protons 
+                                    % LEM: Local Effect Model for carbon ions
+% retrieve bio model parameters
+pln.bioParam = matRad_bioModel(pln.radiationMode,quantityOpt, modelName);
+
+
+% save meta data in machine
+machine.meta.radiationMode = 'protons';
+machine.meta.dataType = 'singleGauss';
+machine.meta.created_on = date;
+machine.meta.created_by = 'Paul Anton Meder';
+machine.meta.SAD = (2218 + 1839) / 2;
+machine.meta.BAMStoIsoDist = 420.0;
+machine.meta.machine = 'Generic';
+machine.meta.LUT_bxWidthminFWHM = [1, Inf; 8 ,8];
+machine.meta.fitAirOffset = 420.0;
+
+
 %% generate steering file
-stf = matRad_generateStf(ct,cst,pln);
- 
+stf = matRad_generateStfSinglePencilBeam(ct,cst,pln,machine.meta.SAD);
```

Comment:
If you use the dummy machine file as suggested, you don#t need the extra SAD argument.

Reply:
Fixed this using the dummy file.